pytezos-tests/pytezos_contract_tester.py

```diff
@@ -257,6 +259,19 @@ def merch_dispute(
 
     return {"status": status, "level": level, "op_info": op_info}
 
+def sign_mutual_close(
+merch_acc,
+contract_id,
+customer_balance, merchant_balance
+):
+    # Merchant pytezos interface
+    merch_py = pytezos.using(key=merch_acc, shell=uri)
+    ty = MichelsonType.match(michelson_to_micheline("pair (pair bls12_381_fr string) (pair address (pair mutez mutez))"))
+    packed = ty.from_python_object((channel_id, 'zkChannels mutual close', contract_id, customer_balance, merchant_balance)).pack(legacy=True).hex()
```

Comment:
looks good! what's the significance of the `legacy=True` option when packing the inputs? can we add a short comment to explain?